packages/shared/src/components/containers/FeedTopicCard.tsx

```diff
@@ -31,8 +32,8 @@ const BackgroundLayer = classed(
 const background: Record<OnboardingFiltersLayout, ReactNode> = {
   grid: (
     <>
-      <BackgroundLayer className="-z-1 opacity-64 group-hover:-rotate-12" />
-      <BackgroundLayer className="opacity-24 -z-2 group-hover:-rotate-[24deg]" />
+      <BackgroundLayer className="invisible group-hover:visible -z-1 opacity-64 group-hover:-rotate-12" />
+      <BackgroundLayer className="invisible group-hover:visible opacity-24 -z-2 group-hover:-rotate-[24deg]" />
```

Comment:
Had to add this to make the resting position look normal. Not sure why it works as intended on previous implementations (AuthModal).
![image](https://github.com/dailydotdev/apps/assets/13744167/bb98eac8-c7f3-4f32-a919-80230ddcd0e3)

Reply:
Works now :)